soul-admin/src/main/java/org/dromara/soul/admin/service/impl/UpstreamCheckService.java

```diff
@@ -134,7 +146,13 @@ public static void removeByKey(final String selectorName) {
      */
     public void submit(final String selectorName, final DivideUpstream divideUpstream) {
         if (UPSTREAM_MAP.containsKey(selectorName)) {
-            UPSTREAM_MAP.get(selectorName).add(divideUpstream);
+            List<DivideUpstream> upstreams = UPSTREAM_MAP.getOrDefault(selectorName, Collections.emptyList());
+            Optional<DivideUpstream> exists = upstreams.stream().filter(item -> StringUtils.isNotBlank(item.getUpstreamUrl()) && item.getUpstreamUrl().equals(divideUpstream.getUpstreamUrl())).findFirst();
+            if (!exists.isPresent()) {
+                UPSTREAM_MAP.get(selectorName).add(divideUpstream);
```

Comment:
Do we really need to call `UPSTREAM_MAP.get(selectorName)` again?

Reply:
this is to deal line too long.
the 152 is forgot to revise it.

Reply:
thanks